src/components/header/_header.scss

```diff
@@ -110,29 +109,41 @@
 
     &--neutral & {
         &__top {
-            background: var(--ons-color-header-masthead-neutral);
+            .ons-icon--logo {
+                display: block;
+                .ons-icon--logo__group--text,
+                .ons-icon--logo__group--primary {
+                    fill: var(--ons-color-black);
+                }
+            }
+        }
+
+        &__org-logo-link:focus {
+            outline: none;
+            background-color: var(--ons-color-focus);
+            .ons-icon--logo {
+                color: var(--ons-color-text-link-focus);
+            }
```

Comment:
This can be restructured so that we aren't repeating so much css. If you use the syntax like on line 207 you should be able to set just the specific things for neutral and internal because I think only a couple of things are different between the two